lang/cs/Org.Apache.REEF.Network/packages.config

```diff
@@ -18,10 +18,49 @@ specific language governing permissions and limitations
 under the License.
 -->
 <packages>
+  <package id="Microsoft.Extensions.Caching.Abstractions" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Caching.Memory" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Options" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Primitives" version="1.1.0" targetFramework="net451" />
```

Comment:
> I did put a jira message exactly like you specified above in my checkin, did that not show up?

Doug's comment wasn't about the jira, but about the *commit* message. We have a guide how to write those [here](https://cwiki.apache.org/confluence/display/REEF/Commit+Messages), but you can also get a a good idea of what they look like from `git log` or the commit log [here on GitHub](https://github.com/apache/reef/commits/master).